core/api/src/graphql/public/types/object/quiz.ts

```diff
@@ -11,6 +13,7 @@ const Quiz = GT.Object({
       description: "The reward in Satoshis for the quiz question",
     },
     completed: { type: GT.NonNull(GT.Boolean) },
+    notBefore: { type: Timestamp },
```

Comment:
is it necessary to return this?

Reply:
yup returning a timeframe at which quiz can be claim it's the whole idea behind this PR